src/Parsers/ASTAlterQuery.cpp

```diff
@@ -611,8 +611,16 @@ void ASTAlterQuery::formatQueryImpl(const FormatSettings & settings, FormatState
 
     FormatStateStacked frame_nested = frame;
     frame_nested.need_parens = false;
-    frame_nested.expression_list_always_start_on_new_line = true;
-    static_cast<ASTExpressionList *>(command_list)->formatImplMultiline(settings, state, frame_nested);
+    if (settings.one_line)
+    {
+        frame_nested.expression_list_prepend_whitespace = true;
```

Comment:
`frame_nested.expression_list_always_start_on_new_line = settings.one_line` is shorter, but it doesn't really matter